src/lib.rs

```diff
@@ -283,17 +230,21 @@ impl RendererConfig<'_, '_> {
             fragment_shader: Some(fragment_shader),
         }
     }
+}
 
+impl Default for RendererConfig<'_, '_> {
     /// Create a new renderer config with precompiled default shaders outputting linear color.
     ///
     /// If you write to a Bgra8UnormSrgb framebuffer, this is what you want.
-    pub fn new() -> RendererConfig<'static, 'static> {
+    fn default() -> Self {
```

Comment:
We shouldn't get rid of the new impl, it's generally good to have a new impl default just directly calls.

Reply:
Yeah I was not sure about that but makes sense. I also replaced some struct names where possible with Self as that seems to be the way clippy likes it.